src/main/java/io/temporal/samples/dsl/SampleActivities.java

```diff
@@ -0,0 +1,77 @@
+package io.temporal.samples.dsl;
+
+import io.temporal.activity.Activity;
+import io.temporal.activity.ActivityInterface;
+import io.temporal.activity.ActivityMethod;
+
+public class SampleActivities {
+  @ActivityInterface
+  public interface SampleActivities1 {
+    @ActivityMethod
+    String getInfo1();
+  }
+
+  @ActivityInterface
```

Comment:
Ryalnd method that you can pass a prefix to add the activity name to the register method, but neither the activityInterface or the activitymethod allows for this.  Any suggestion on how this should be implemented.

I want to be able to have the same method name for different interfaces